src/main/java/tp/cap5buddy/todolist/ParserUtilTodoList.java

```diff
@@ -0,0 +1,140 @@
+package tp.cap5buddy.todolist;
```

Comment:
We could use a common ParserUtil class to implement parsing methods which can be used across ModuleList, ContactList and TodoList in the future. For now, this is a sound implementation.

Reply:
Yes, I will migrate all the methods to the ParserUtil class.